group_vars/all

```diff
@@ -25,3 +25,10 @@ validate_certs: "yes"
 # env:
 # service_vip:
 # control_interface:
+
+# should mons also run osds
+mon_osd_colocation_allowed: "true"
+
+# OSDs could be run without netplugin on them
+# osd_contiv_network_include controls this behaviour
+osd_contiv_network_include: "true"
```

Comment:
@mapuri.. yes, consolidating under a common var would be good. But I think we should specify exclusion. So, everything is included by default, unless the an exclusion is specified. for example the below would specify not to run mon and osd on the given host. It can be one large string of comma separated values.

host-capability => “not_mon, not_osd"
`when: host-capability | match(*not_osd*)`